src/test/java/uk/gov/hmcts/reform/sandl/snlapi/controllers/SecurityControllerTest.java

```diff
@@ -77,6 +129,13 @@ public void shouldReturnUserDetails_whenLoggedIn() throws Exception {
             .header("Authorization", "Bearer " + token.getAccessToken())
             .contentType(MediaType.APPLICATION_JSON))
             .andExpect(status().isOk())
-            .andExpect(MockMvcResultMatchers.jsonPath("$.username").value(USERNAME));
+            .andExpect(MockMvcResultMatchers.jsonPath("$.username").value(OFFICER1));
+    }
+
+    private void checkUser(String username, String encodedPassword, boolean isEnabled, boolean isResetRequired) {
+        User user = userRepository.findByUsername(username);
```

Comment:
Yes. checkUser was a private helper method to make sure the data coming from UserRepository is correct, before testing the SecurityController class itself (otherwise you can end up with tests passing when they shouldn't i.e. you want to differentiate between username not found, and user credentials expired etc)